examples/rustls-mtls/src/lib.rs

```diff
@@ -1,22 +1,19 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This example is quite a mess at this point. We shouldn't be forcing applications to write their own TLS providers just to use mTLS. Can you open an issue so we can clean it all up? Or I guess it's being addressed in #2129?